### DISCUSSION TITLE
Can I use a verification document in another language?

### DISCUSSION THREAD

**Discussion by thund-x** [community/community#44354] [GitHub Education] [answered]
Hello! I'm a student of a Greek university. We have an academic ID card which is similar to [ISIC cards](https://www.isic.org/) and includes all required information for verification, but they are in Greek, and on the back side of the card as well. 

Based from similar discussions, the verification system is automatic. Is it fine with the information being in Greek? If not, are there any workarounds?

Can I take a photo of both sides and merge them into one photo? Thanks!

**Comment by alexeytatarchuk** [comment, marked as answer]
Yes, it is possible, but one photo must be accompanied by a document with an English translation

**Comment by thund-x** [comment]
Can I provide a handwritten translation?

**Comment by bzkarim250** [comment]
It's best to check the requirements of the system you're using, and if they require separate images, you can use a photo editing software to merge them into one image. However, make sure that all the information are clearly visible and legible in the merged image.

**Comment by thund-x** [comment]
Got it, thanks.
